client/blocks/product-purchase-features-list/jetpack-activity.jsx

```diff
@@ -0,0 +1,34 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+import { getSelectedSite } from 'state/ui/selectors';
+
+const JetpackSiteActivity = ( { site, translate } ) => (
```

Comment:
A question regarding component naming: if we name it `JetpackSiteActivity`, it looks like it applies only for Jetpack sites, and that doesn't seem to be the case. Do we want to change the component name so it reflects the more broad use case for the activity card?